quickstart/speech-translation/csharp-uwp/helloworld/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("helloworld")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("helloworld")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```

Comment:
>2018")] [](start = 43, length = 7)

fix where it applies

Reply:
I don't have any real experience in UWP development, so I don't follow this comment. What's the request? The file looks identical to what's provided for the recognition sample.